source/sass/glyphs.scss

```diff
@@ -39,6 +39,14 @@
       top: 2px;
     }
   }
+
+  &.form-error-glyph {
+    &::before {
+      width: 13px;
+      height: 20px;
```

Comment:
This glyph itself is 13x13px so let's make sure height is set to match that value.